cmd/agent/agent.go

```diff
@@ -148,6 +167,29 @@ func loop(c *cli.Context) error {
 		return err
 	}
 
+	agentID, err = client.RegisterAgent(ctx, platform, engine.Name(), version.String(), parallel)
+	if err != nil {
+		return err
+	}
+
+	log.Debug().Msgf("agent registered with id %d", agentID)
```

Comment:
To be more consistent 
```suggestion
	log.Debug().Msgf("Agent registered with ID %d", agentID)
```